.github/workflows/pr_test.yml

```diff
@@ -4,48 +4,50 @@ jobs:
   lint:
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout code
+    - name: "Checkout code"
```

Comment:
The changes to this file can be summarised as follows...

1. Add `name` field if missing.
2. Reword some `name` fields.
3. Tweak Golang caching.
4. Add Rust caching.